classes/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
Great job so far. Please add a method for adding rentals since it has a `many-to-many` relationship and it's one of the project requirements.
![image](https://github.com/Mike47ip/OOP-Library/assets/101523582/1155b22b-bc26-4d0f-af54-282ca54a1aa0)

You can refer to this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more.  Kindly, add it